src/modules/editTodo.js

```diff
@@ -0,0 +1,44 @@
+import { createList } from './displayList.js';
+import remove from '../assets/images/delete.svg';
+
+const trash = new Image();
+trash.src = remove;
+export default function EditTodo(toDo) {
+  const editButtons = document.querySelectorAll('.dots');
+  for (let i = 0; i < editButtons.length; i += 1) {
```

Comment:
Please try to use a module in this part and call the methods here, this way your code will be cleaner.